README.md

```diff
@@ -2,6 +2,38 @@
 
 Smoke tests for kubernetes clusters
 
-See `makefile` for details of how to run.
-
 Tests tagged 'cluster:live-1' will only work if the current kubernetes config points to the live-1 cluster.
+
+## How to run smoke-tests
+The smoke tests will run inside a docker container against your current Kubernetes context (you can see your context by running `kubectl config get-contexts`).
+
+### Pre requisites
+- git
+- docker
+- `cluster-admin` access to a Cloud Platform cluster
+
+### Build or pull the image
+You'll need to have pulled the image locally, you can do this with the following:
```

Comment:
The way this reads is a little bit confusing. I'd put the "pull" section immediately below this heading.

Reply:
latest commit should've sorted this.